base/logallocator.go

```diff
@@ -14,10 +15,11 @@ import (
 type LogAllocator struct {
 	recordPool   *sync.Pool         // pool of *LogRecord
 	backbufPools util.BytesPoolBy2n // pools of the backing buffers of LogRecord(s), i.e. pools of raw input copies
+	outputCount  int                // the amount of outputs and consecutively initial refcount
```

Comment:
Maybe this is better named something like `initialRefCount`, since the allocator doesn't need to know about output?